packages/coinstac-ui/scripts/build.js

```diff
@@ -10,7 +10,6 @@ const options = {
   overwrite: true,
   platform: 'all',
   prune: true,
-  version: '1.1.1',
```

Comment:
This is automatically detected by `packager`, setting it explicitly is just another thing to update